File: Aula_03/listapoo01_ex03.cpp
#include <iostream>

using namespace std;

class ContaBancaria {
    public:
        string titular = "";
        string numero = "";
        double saldo = 0;
        void depositar(double v) {
            this->saldo += v;
        }
        void sacar(double v) {
            this->saldo -= v; 
        }        
};

int main() {
    ContaBancaria *x = new ContaBancaria();
    x->titular = "Eduardo";
    x->numero = "123";
    cout << x->saldo << endl;
    x->depositar(100);
    cout << x->saldo << endl;
    x->sacar(20);
    cout << x->saldo << endl;
    delete x;
    return 0;
}

File: Aula_03/listapoo01_ex03.py
class ContaBancaria:
    def __init__(self):
        self.titular = ""
        self.numero = ""
        self.saldo = 0
    def depositar(self, v):
        self.saldo += v
    def sacar(self, v):
        self.saldo -= v    

x = ContaBancaria()
x.titular = "Eduardo"
x.numero = "123-X"
print(x.saldo)
x.depositar(100)
print(x.saldo)
x.sacar(20)
print(x.saldo)

